src/data/grammar.json

```diff
@@ -40,15 +41,18 @@
         "The Importance of [_thing:#card#,#cardCategory.s#,#creatureTypes.s#,#gameConcept#]#_thing# in [_where:#format#,#recentCardSet#]#_where#",
         "#pros# Don't realize that #tech.s# #verbDefeat# #obstacle.capitalize#",
         "The Secret to #improvementGoal.capitalizeAll# is #deckChange.capitalize#",
-        "In today's Magic Story: #character#'s #adj# history with #character# finally revealed"
+        "In today's Magic Story: #character#'s #adj# history with #character# finally revealed",
+        "Video: MTG – How I would improve #thing# in Magic: the Gathering, Part #num2-9#",
```

Comment:
The redundancy is intentional, mirroring YouTube video names made this way for search results -- see TheManaSource as example. Similarly I wasn't sure if we should attach real names to this since we are making this "author" look pretty bad.